lib/index.js

```diff
@@ -405,14 +414,16 @@ api.updatePublicKey = (actor, publicKey, callback) => {
  * @param publicKeyId the ID of the publicKey to revoke.
  * @param callback(err, key) called once the operation completes.
  */
-api.revokePublicKey = (actor, publicKeyId, callback) => {
+api.revokePublicKey = ({actor, publicKeyId}, callback) => {
   if(actor === undefined) {
     return callback(new BedrockError(
       'Permission denied; no actor specified.',
-      'PermissionDenied',
+      'NotAllowedError',
```

Comment:
Are we replacing `PermissionDenied` with `NotAllowedError` in new code?

Reply:
That was my plan, what do you think?

Reply:
That's fine. I just want to make sure we're on the same page. I'll update bedrock-identity to do the same.